Apps/VisionOS/VisionOS/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+import CioTracking
+import UIKit
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication
+            .LaunchOptionsKey: Any]? = nil
+    ) -> Bool {
+        /*
+
+         How do I get the values for the siteId and apiKy?
+         You can find/create them here: https://fly.customer.io/settings/api_credentials
+         For more information about workspaces checkout these
+         docs: https://customer.io/docs/accounts-and-workspaces/workspaces/
+         */
+
+        let workspaceSettings = AppState.shared.workspaceSettings
+        if workspaceSettings.isSet() {
+            CustomerIO.initialize(
+                siteId: workspaceSettings.siteId,
+                apiKey: workspaceSettings.apiKey,
+                region: workspaceSettings.region
+            ) { config in
+                // Debug config just to make the demo
+                // easier. You can learn more about these configs
+                // here: https://customer.io/docs/sdk/ios/getting-started/#configuration-options
+                config.backgroundQueueMinNumberOfTasks = 1
+                config.backgroundQueueSecondsDelay = 0
```

Comment:
Despite the fact that we provide these configurations (`backgroundQueueMinNumberOfTasks`, `backgroundQueueMinNumberOfTasks` to be specific), [we do not recommend our users to update these values](https://customer.io/docs/sdk/ios/getting-started/#configuration-options) as it might affect the performance of the app and battery life of the device.

Just thinking out loud if we would want to update these values to minimum for the demo app ? 🤔

Reply:
Also wondering that we might have to do additional efforts on this after we release CDP. There is a huge change when migrating from Journeys to CDP !

Reply:
Not that much huge change, IMO. Just updating the callistes to use DataPipeline APIs instead of Tracking